tailwind.config.js

```
@@ -0,0 +1,82 @@
+ /** @type {import('tailwindcss').Config} */
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)